doc/ref/configuration/master.rst

```diff
@@ -180,7 +180,7 @@ The path to the master's configuration file.
 ``pki_dir``
 -----------
 
-Default: ``/etc/salt/pki/master``
+Default: ``<LIB_STATE_DIR>/pki/master``
```

Comment:
Could you add more information about what LIB_STATE_DIR is? A user would not know what this is without reading the code or changelog. And include that by default it does indeed default to /etc/salt/ if nothing specified with `--salt-lib-state-dir`. 
This also applies to the docs in doc/ref/configuration/minion.rst